### DISCUSSION TITLE
6.12 is a roadblock

### DISCUSSION THREAD

**Discussion by n0xena** [demizer/archzfs#559] [General]
so I just did my daily update I noticed 6.12 got released - hence I tried to build 2.2.6 and 2.3.0-rc3 against it - and failed:
2.2.6 gets thru creating the makefile and starts to compile the source but somewhere fails during compilation
2.3.0-rc3 already fails creating the makefile due to kernel version enforcement
over at my fork I currently have 2.2.6 and 2.3.0-rc3 for 6.11.9 and lts 6.6.62
currently I'm using 2.3.0-rc3 bare metal on my system with 6.11.9 without any issues so far
I also updated my request at upstream https://github.com/openzfs/zfs/discussions/16574
from my daily tech news site I read that 6.12 will become the new lts as it's the last release for this year - which breaks the "stick to lts for support" when that's the same as current
in the main discussion this was already mentioned - but I haven't read a reply how to deal with this now

**Comment by n0xena** [comment]
as an update: https://github.com/openzfs/zfs/discussions/16574#discussioncomment-11361198 got me a hint how to pass the kernel version enforcement and I was able to successfully build [zfs-linux-rc-2.3.0_rc3_6.12.1.arch1.1-1-x86_64.pkg.tar.zst](https://github.com/n0xena/archzfs/releases/tag/2.3.0-rc3_6.12.1)

**Comment by Soulsuke** [comment]
I agree this breaks the "stick to lts" approach, maybe we should rephrase how this repo works: no matter if you use stock or lts, as soon as a new version gets released you may have to resort to partial upgrades until openzfs catches up. Of course this happens more often with mainline, but lts takes the hit sooner or later...

And I have to say, [this one](https://github.com/openzfs/zfs/issues/11097) makes me worried about 6.12 :\

**Comment by n0xena** [comment]
https://github.com/openzfs/zfs/releases/tag/zfs-2.2.7
